nova_vm/src/ecmascript/builtins/text_processing/string_objects/string_prototype.rs

```diff
@@ -243,6 +245,113 @@ impl Builtin for StringPrototypeIterator {
     const BEHAVIOUR: Behaviour = Behaviour::Regular(StringPrototype::iterator);
 }
 
+/// ### [22.1.3.17.1 StringPaddingBuiltinsImpl ( O, maxLength, fillString, placement )](https://tc39.es/ecma262/#sec-stringpaddingbuiltinsimpl)
```

Comment:
**nitpick:** I've usually put these sorts of "helper" functions to the bottom of the file, after the impls and so on.